### PR TITLE
Check both 32-bit and 64-bit registry when looking for the uninstaller

### LINKED ISSUE #5845: [🐛 | Bug] Artefact `_psutil_windows.cp39-win_amd64.pyd` from a previous parsec installation can cause psutil to fail
Sentry Issue: [PARSEC-R53](https://scille.sentry.io/issues/4658827113/?referrer=github_integration)

```
JobResultError: ('crashed', {'exc': ImportError("version conflict: 'C:\\\\Program Files (x86)\\\\Parsec Cloud\\\\psutil\\\\_psutil_windows.cp39-win_amd64.pyd' C extension module was built for another version of psutil (5.9.3 instead of 5.9.4); you may try to 'pip uninstall psutil', manually remove C:\\Program Files (x86)\\Parsec Cloud\\psutil\\_psutil_windows.cp39-win_amd64.pyd or clean the virtual env somehow, then reinstall"), 'info': 'Unexpected error: ImportError("version conflict: \'C:\\\\\\\\Program Files (x86)\\\\\\\\Parsec Cloud\\\\\\\\psutil\\\\\\\\_psutil_windows.cp39-win_amd64.pyd\' C extension module was built for another version of psutil (5.9.3 instead of 5.9.4); you may try to \'pip uninstall psutil\', manually remove C:\\\\Program Files (x86)\\\\Parsec Cloud\\\\psutil\\\\_psutil_windows.cp39-win_amd64.pyd or clean the virtual env somehow, then reinstall")', 'origin': None})
(5 additional frame(s) were not displayed)
...
  File "parsec\core\mountpoint\manager.py", line 447, in mountpoint_manager_factory
  File "parsec\core\win_registry.py", line 230, in cleanup_parsec_drive_icons
  File "parsec\core\win_registry.py", line 45, in get_psutil
  File "pyimod02_importers.py", line 352, in exec_module
  File "__init__.py", line 241, in <module>

Unhandled error while logging in
```

### PR DESCRIPTION
**Pull request by vxgmichel**
Fix #5845

### CODE REVIEW

File: packaging/windows/installer.nsi [hunk 1]
```diff
@@ -205,9 +194,51 @@ Function .onInit
...
       ExecWait '"$R0" /S _?=$R1'
+FunctionEnd
 
-    done:
+Function checkUninstaller
+  ; Check either 32-bit or 64-bit registry for parsec uninstaller
+  ReadRegStr $R0 ${PROGRAM_UNINST_ROOT_KEY} \
+    "${PROGRAM_UNINST_KEY}" \
+    "UninstallString"
+    StrCmp $R0 "" done
```
Comment (FirelightFlagboy) [inline packaging/windows/installer.nsi:204]:
```suggestion
  StrCmp $R0 "" done
```

I'm not sure, but it fells like the `StrCmp` is at the wrong indentation level :thinking:

File: packaging/windows/installer.nsi [hunk 2]
```diff
@@ -205,9 +194,51 @@ Function .onInit
...
+      /SD IDOK IDOK uninstall IDCANCEL _abort
+
+  ; Abort in case the user clicked "cancel"
+  _abort:
+    Abort
+
+  ; Run the uninstaller
+  uninstall:
+    Call runUninstaller
+    Goto done
```
Comment (FirelightFlagboy) [inline packaging/windows/installer.nsi:228]:
```suggestion
  uninstall:
    Call runUninstaller
    Call cleanup
    Goto done
```

Once the `runUninstaller` routine has finished, should we cleanup the register values ?

File: packaging/windows/installer.nsi [hunk 3]
```diff
@@ -265,14 +296,11 @@ FunctionEnd
...
 # --- Installation sections ---
-!define PROGRAM_UNINST_KEY "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PROGRAM_NAME}"
-!define PROGRAM_UNINST_ROOT_KEY "HKLM"
-!define PROGRAM_UNINST_FILENAME "$INSTDIR\uninstall.exe"
 
 BrandingText "${PROGRAM_NAME} Windows Installer v${INSTALLER_SCRIPT_VERSION}"
 Name "${PROGRAM_NAME} ${PROGRAM_VERSION}"
 OutFile "${BUILD_DIR}\${INSTALLER_FILENAME}"
-InstallDir "$PROGRAMFILES\Parsec Cloud"
+InstallDir "$PROGRAMFILES64\${PROGRAM_NAME}\${PROGRAM_VERSION}"
```
Comment (FirelightFlagboy) [inline packaging/windows/installer.nsi:303]: The previous uninstaller take care of removing the parsec files under `$PROGRAMFILES` ?

File: packaging/windows/installer.nsi (hunk 2 above)
Comment (vxgmichel) [reply packaging/windows/installer.nsi:228]: The uninstaller is meant to cleanup those values, so I chose to cleanup the registry only if the uninstaller no longer exists. It's kind of arbitrary tho

File: packaging/windows/installer.nsi (hunk 3 above)
Comment (vxgmichel) [reply packaging/windows/installer.nsi:303]: Exactly

Comment (FirelightFlagboy) [reply packaging/windows/installer.nsi:303]: :+1:

File: packaging/windows/installer.nsi (hunk 2 above)
Comment (FirelightFlagboy) [reply packaging/windows/installer.nsi:228]: Ok, so as you said, the uninstaller remove the register value then the cleanup just after is redundant

File: packaging/windows/installer.nsi (hunk 1 above)
Comment (vxgmichel) [reply packaging/windows/installer.nsi]: fixed!

**Review by touilleMan — APPROVED** [review]

**Pull request merged**